Bastion Guide Update

FC-Austar opened the pull request:
Amended requirements section to include armor comp requirement minimum.
Amended Site information to reflect most current usage of bastion (as per disco FC voting).
Added "Additional Tips" section to include some relevant bastion information.

FC-Austar committed:
Bastion Guide Update

Amended requirements section to include armor comp requirement minimum.
Amended Site information to reflect most current usage of bastion (as per disco FC voting).
Added "Additional Tips" section to include some relevant bastion information.

TvdW commented on frontend/src/Pages/Guide/guides/bastion/guide.md:
```diff
@@ -19,32 +19,41 @@ Bastion is for experienced incursion runners only. This means to use it you MUST
 4.  If someone dies on-grid you are to turn Bastion off for the rest of the site.
 5.  Full Hybrid or Amulet pod is required for SRP.
-6.  Basic tank skills.
+6.  Basic tank skills. (Armor comps to level 4)
```
This may be misleading - we also require mechanics and hull upgrades at a certain level. For style we should also always write 'armor compensation skills' out in full.

FC-Austar replied:
perhaps this just needs rewording, maybe something like "All tank skills to basic level for your chosen ship or higher"

TvdW commented on frontend/src/Pages/Guide/guides/bastion/guide.md:
```diff
@@ -19,32 +19,41 @@ Bastion is for experienced incursion runners only. This means to use it you MUST
+6.  Basic tank skills. (Armor comps to level 4)
 7.  Be aware when using the micro jump drive of your direction, and of bumps. If you jump poorly and die due to being out of logi range, this is pilot error and WILL NOT be covered under SRP. (Being bumped by another fleet member is also not covered under SRP). If you're not confident, DO NOT USE IT.
 
+# Site guidance
```
Heading-1 is for page titles only, you should probably make this a h-2 and lower the rest to h-3

FC-Austar replied:
I actually have no idea how to do this, is it as simple as adding more #'s to the front of the headings?

TvdW replied:
Yup!

TvdW commented on frontend/src/Pages/Guide/guides/bastion/guide.md:
```diff
@@ -19,32 +19,41 @@ Bastion is for experienced incursion runners only. This means to use it you MUST
 3.  You do not Bastion until aggro is established on one target at the start of a wave/site.
 4.  If someone dies on-grid you are to turn Bastion off for the rest of the site.
 5.  Full Hybrid or Amulet pod is required for SRP.
```
Not part of your change but it raised a question recently - does it require the "full pod" or just 1-6?

FC-Austar replied:
will check with council one i find one and let you know

FC-Austar committed:
Revision 2 to bastion guide

Amended skills wording to better reflect tank requirements as a whole.
Amended "additional tips" heading to be smaller and thus in line with formatting.

FC-Austar committed:
Revision 3

Amended requirement for SRP to needing amulets 1 through 5 as per confirmation by Joana (Council)

TvdW committed:
Fix bastion guide headings

TvdW committed:
Bastion guide: style the "Additional tips" section

TvdW committed:
Fixup indentation of bullet list
